app/views/support_interface/performance/feature_metrics_dashboard.html.erb

```diff
@@ -14,9 +14,7 @@
   <div class="govuk-grid-column-one-half">
```

Comment:
probably should add a _last generated at ..._ message on this view somewhere.

Reply:
Yeah definitely, was thinking a 'last generated at' and 'next update expected' would be useful.